chainer/utils/conv.py

```diff
@@ -5,6 +5,24 @@
 
 
 def get_conv_outsize(size, k, s, p, cover_all=False, d=1):
+    """Calculate output size of convolution.
```

Comment:
Ditto. the same points as `get_deconv_outsize`

Reply:
Thanks, fixed the typo.